js-exercises/better-string-lib/betterStringLib.js

```diff
@@ -0,0 +1,16 @@
+const equal = (str1, str2) => !str1.localeCompare(str2);
+const normalize = (str) => str.normalize("NFC");
+function reverse(str) {
+  if (!str.length) return "";
```

Comment:
Validate that str variable has any value or not. Let say str has 'undefined' or 'null' value. in this case, your code will break because while javascript executes str.length line, you will get an error called "can not read a property length of undefined or null". So, please put extra validation if possible.